--- /workspaces/task-director/server/server/settings.py	2022-01-05 12:24:37.968308859 +0000
+++ /workspaces/task-director/server/server/settings_copy.py	2022-01-05 12:25:18.211000000 +0000
@@ -135,12 +135,12 @@
         #     ],
         # },
         ### Method 2: Via local Redis => `docker run -p 6379:6379 -d redis:5`
-        "BACKEND": "channels_redis.core.RedisChannelLayer",
-        "CONFIG": {
-            "hosts": [("172.17.0.1", 6379)],
-        },
+        # "BACKEND": "channels_redis.core.RedisChannelLayer",
+        # "CONFIG": {
+        #     "hosts": [("172.17.0.1", 6379)],
+        # },
         ### Method 3: Via In-memory channel layer
-        # "BACKEND": "channels.layers.InMemoryChannelLayer"
+        "BACKEND": "channels.layers.InMemoryChannelLayer"
     },
 }
 
